wire-ios/Wire-iOS/Sources/UserInterface/Settings/Backup/BackupPasswordValidator.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import WireSettingsUI
+
+struct BackupPasswordValidator: BackupPasswordValidatorProtocol {
+
+    func isValid(password: String) -> Bool {
+        guard !password.isEmpty else {
```

Comment:
Isn't this case already covered by `PasswordRuleSet`?
Maybe we can get rid of the validation logic here.

Reply:
We use `PasswordRuleSet` for login too. The empty password for login isn't allowed, but it's allowed for backup

Reply:
Hm, still it would be nice to have all the actual validation code in one place (e.g. in the type `PasswordRuleSet`).
What about creating a separate method there?
Or leave a comment here.
optional